File: sdkwork-backend-sdk-go/types/plus_user_profile_vo.go
package types

// 用户个人资料VO
type PlusUserProfileVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	FaceImage ImageMediaResource `json:"faceImage"`
	FaceVideo VideoMediaResource `json:"faceVideo"`
	Username string `json:"username"`
	Nickname string `json:"nickname"`
	Email string `json:"email"`
	Phone string `json:"phone"`
	Gender string `json:"gender"`
	Status string `json:"status"`
	CountryCode string `json:"countryCode"`
	ProvinceCode string `json:"provinceCode"`
	CityCode string `json:"cityCode"`
	DistrictCode string `json:"districtCode"`
	Address string `json:"address"`
	Bio string `json:"bio"`
	BirthDate string `json:"birthDate"`
	SocialInfoList SocialInfoList `json:"socialInfoList"`
}

File: sdkwork-backend-sdk-csharp/Models/PlusUserProfileVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusUserProfileVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public ImageMediaResource? FaceImage { get; set; }
        public VideoMediaResource? FaceVideo { get; set; }
        public string? Username { get; set; }
        public string? Nickname { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? Gender { get; set; }
        public string? Status { get; set; }
        public string? CountryCode { get; set; }
        public string? ProvinceCode { get; set; }
        public string? CityCode { get; set; }
        public string? DistrictCode { get; set; }
        public string? Address { get; set; }
        public string? Bio { get; set; }
        public string? BirthDate { get; set; }
        public SocialInfoList? SocialInfoList { get; set; }
    }
}
